packages/salesforcedx-vscode-core/package.json

```diff
@@ -121,14 +121,14 @@
         {
           "id": "metadata",
           "name": "Metadata",
-          "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking"
+          "when": "sfdx:project_opened && sfdx:enable_org_browser"
```

Comment:
When I first started working on this, it took me a while to figure out that "sfdx:default_username_has_no_change_tracking" meant it was for a developer edition org, and "sfdx:default_username_has_change_tracking" meant it was a scratch org.  I renamed these functions (used for the where clauses) to what they enable/disable (sfdx:enable_deploy_and_retrieve_commands and sfdx:enable_push_and_pull_commands), and also added sfdx:enable_org_browser)